src/Looplex.Protocols.HTTP/README.md

```
@@ -0,0 +1,538 @@
+    },
+    "emails": [{
+      "value": "john.doe@example.com",
+      "primary": true
+    }]
+  }'
+```
+
+#### Retrieve User
+```bash
+curl -X GET "http://localhost:54075/Users/123e4567-e89b-12d3-a456-426614174000" \
+  -H "Accept: application/scim+json"
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)